Packs/Devo/Integrations/Devo_v2/Devo_v2.py

```diff
@@ -641,27 +710,66 @@ def write_to_lookup_table_command():
     return [entry]
 
 
-''' EXECUTION CODE '''
-try:
-    if ALLOW_INSECURE:
-        os.environ['CURL_CA_BUNDLE'] = ''
-        os.environ['PYTHONWARNINGS'] = 'ignore:Unverified HTTPS request'
-    handle_proxy()
-    if demisto.command() == 'test-module':
-        check_configuration()
-        demisto.results('ok')
-    elif demisto.command() == 'fetch-incidents':
-        fetch_incidents()
-    elif demisto.command() == 'devo-run-query':
-        demisto.results(run_query_command())
-    elif demisto.command() == 'devo-get-alerts':
-        demisto.results(get_alerts_command())
-    elif demisto.command() == 'devo-multi-table-query':
-        demisto.results(multi_table_query_command())
-    elif demisto.command() == 'devo-write-to-table':
-        demisto.results(write_to_table_command())
-    elif demisto.command() == 'devo-write-to-lookup-table':
-        demisto.results(write_to_lookup_table_command())
-
-except Exception as e:
-    return_error('Failed to execute command {}. Error: {}'.format(demisto.command(), str(e)))
+def main():
+    ''' EXECUTION CODE '''
+    try:
+        if ALLOW_INSECURE:
+            os.environ['CURL_CA_BUNDLE'] = ''
+            os.environ['PYTHONWARNINGS'] = 'ignore:Unverified HTTPS request'
+        handle_proxy()
+        if demisto.command() == 'test-module':
+            check_configuration()
+            demisto.results('ok')
+        elif demisto.command() == 'fetch-incidents':
+            fetch_incidents()
+        elif demisto.command() == 'devo-run-query':
+            OFFSET = 0
+            items_per_page = int(demisto.args()['items_per_page'])
+            if items_per_page <= 0:
+                raise ValueError(
+                    'items_per_page should be a positive non-zero value.')
+            total = 0
+            demisto.results(run_query_command(OFFSET, items_per_page))
+            total = total + COUNT_SINGLE_TABLE
+            while COUNT_SINGLE_TABLE == items_per_page:
+                OFFSET = OFFSET + items_per_page
+                total = total + COUNT_SINGLE_TABLE
+                demisto.results(run_query_command(OFFSET, items_per_page))
+        elif demisto.command() == 'devo-get-alerts':
+            OFFSET = 0
+            items_per_page = int(demisto.args()['items_per_page'])
```

Comment:
same

Reply:
Fixed in commit cb16ecc9973bea2c57742a8a52ee437c6bf1c141